Gemfile

```diff
@@ -21,6 +21,7 @@ gem 'dalli' # memcached
 gem 'parallel'
 
 gem 'google-api-client'
+gem 'sprockets-derailleur' # Multi-cpu assets precompile
```

Comment:
Is sprockets-derailleur an essential part of this change or just an optimization?  If it's non-critical, it might be preferable to factor this into a separate PR, to optimize for getting your key changes in and not having to take a detour to fix any unexpected regressions (e.g. due to concurrency bugs) which might be exist with sprockets-derailleur.